src/migrations/20201205200638-create-ttaplan.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Ttaplans', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      granteeId: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        references: {
+          model: {
+            tableName: 'Grantees',
+          },
+          key: 'id',
+        },
+        onUpdate: 'CASCADE',
+        onDelete: 'CASCADE',
+      },
+      // grant: {
+      //   type: Sequelize.STRING,
+      //   allowNull: false,
+      // },
+      grantId: {
```

Comment:
Does the `ttaplans` table need a foreign key to both `grantees` and `grants`? I would think it would only need a FK to `grants` but may be missing something!

Reply:
Right, it needs grants FK with the current design. How about we address potentially removing the grantee FK in a different PR?
Since things are still fluid at this point, this would also cover a case where we would need to have a many to many relationship between grants and grantees for some reason.